pkg/constant/fast_confirmation_multiplier.go

```diff
@@ -0,0 +1,72 @@
+package constant
+
+import (
+	"math/big"
+
+	sdkmath "cosmossdk.io/math"
+
+	"github.com/zeta-chain/node/pkg/chains"
+)
+
+var (
+	// DefaultInboundFastConfirmationLiquidityMultiplier is the default ZRC20 liquidity cap multiplier for inbound fast confirmation
+	// The reason of using decimal type is for easy integration into ChainParams, like field `ballot_threshold`
+	DefaultInboundFastConfirmationLiquidityMultiplier = sdkmath.LegacyMustNewDecFromStr("0.00025")
+
+	// InboundFastConfirmationLiquidityMultiplierMap maps chainID to ZRC20 liquidity cap multiplier for inbound fast confirmation.
+	// Fast inbound confirmation is enabled only for chains explicitly listed in this map.
+	InboundFastConfirmationLiquidityMultiplierMap = map[int64]sdkmath.LegacyDec{
```

Comment:
We should avoid using map because iteration is non deterministic and can cause consensus failure in the chain

Reply:
The map is actually for searching only.

Reply:
I'll resolve this. The map is renamed in commit for searching only https://github.com/zeta-chain/node/pull/3551/commits/6596863f4b5045e05799cfd06e5e40543ca2c90f